frontend/public/components/utils/tile-view-page.jsx

```diff
@@ -604,6 +604,7 @@ export class TileViewPage extends React.Component {
         {_.map(filterGroup, (filter, filterName) => {
           const { label, active } = filter;
           return <FilterSidePanel.CategoryItem
+            id={`${groupName}-${filterName.replace(/ /g, '-')}`}
```

Comment:
I'm not sure what values `filterName` can be, but we might want `_.kebabCase` to handle any other special characters that aren't legal element IDs

Reply:
Fixed, thanks.